### ISSUE TITLE
React native 65 and above

### ISSUE THREAD

**Issue by niloufarMakhzani** [zubricky/react-native-android-keyboard-adjust#32]
Would you please change the   compile "com.facebook.react:react-native:+" to   implementation "com.facebook.react:react-native:+" i android>build.gradle file as compile is not working on react native 65 and above

**Comment by AlexTaran** [comment]
Same issue for me, the problem is android gradle plugin version 7.0

https://stackoverflow.com/questions/23796404/could-not-find-method-compile-for-arguments-gradle

This package was updated 4 years ago, isn't it just abandoned? Should we just use an alternative?

For example: https://www.npmjs.com/package/rn-android-keyboard-adjust

**Comment by redouane-boudella** [comment]
> Same issue for me, the problem is android gradle plugin version 7.0
> 
> https://stackoverflow.com/questions/23796404/could-not-find-method-compile-for-arguments-gradle
> 
> This package was updated 4 years ago, isn't it just abandoned? Should we just use an alternative?
> 
> For example: https://www.npmjs.com/package/rn-android-keyboard-adjust

YES DEFINITLY MIGRATE to https://www.npmjs.com/package/rn-android-keyboard-adjust

**Comment by StanislavMayorov** [comment]
> For example: https://www.npmjs.com/package/rn-android-keyboard-adjust

I think this package is much better option. https://www.npmjs.com/package/react-native-avoid-softinput
I tried `rn-android-keyboard-adjust`, but it requires node v16 and installs its own gradle (potential conflicts) without a good reason.
